### DISCUSSION TITLE
Facebook same workaround? is it possible or already exists?

### DISCUSSION THREAD

**Discussion by TeusDev** [baffler/Transparent-Twitch-Chat-Overlay#71] [Ideas]
hi devs, just asking if u guys know or are easy-to-code same workaround for facebook streaming. Thanks in advancd.

**Comment by baffler** [comment]
Found a video that can show you how to set that up. It tells you to copy the Widget URL to OBS but you'd instead put it into my app, so if you want it as the main window, just go to "Chat" in the settings page and at the bottom change the box to the left of "Save" to Custom URL and paste the link there.

If you want the chat as a new window in the app, click "Widgets" in the settings and paste the URL there and click the "Create" button.

https://www.youtube.com/watch?v=Q_ype3niSA8
